.gitignore

```diff
@@ -19,3 +19,4 @@
 /web/config.php
 logs/
 build/
+.idea/
```

Comment:
this is not bundle related, but phpstorm hidden files, you can exclude it in your local gitignore instead of here